locales/pt/json.json

```diff
@@ -137,8 +137,8 @@
     "Choose Type": "Escolher Tipo",
     "Christmas Island": "Ilha Do Natal",
     "City": "Cidade",
-    "Click here to re-send the verification email.": "Click here to re-send the verification email.",
-    "click here to request another": "clique aqui para solicitar outro",
+    "Click here to re-send the verification email.": "Clique aqui para reenviar o e-mail de verificação.",
+    "click here to request another": "Clique aqui para solicitar outro",
```

Comment:
This sentence starts with a small letter in English. Is it correct to start with a capital letter?

Reply:
It ain't correct. Has been fixed with the latest commit. Thanks for the feedback.

Reply:
Thank you